backend/api/keycloak_authentication.py

```diff
@@ -91,7 +96,7 @@ def authenticate(self, request):
 
         keys = self._get_keys().items()
 
-        if len(keys) == 0:
+        if not keys:
```

Comment:
I think it is possible for `keys` to exist but be empty.

Reply:
I'll revert the code